services/lightlevel.md

```diff
@@ -0,0 +1,21 @@
+# Light level
```

Comment:
I suppose we would have a separate luxmeter service for a precise one?

Reply:
yes luxmeter would be a different one, much more high-end